cirq-superstaq/cirq_superstaq/ops/qudit_gates.py

```diff
@@ -504,9 +516,13 @@ def qubit_subspace_op(
 QutritZ2 = QutritZ2PowGate()
 
 
-def custom_resolver(  # pylint: disable=missing-function-docstring
+def custom_resolver(
     cirq_type: str,
 ) -> Optional[Type[cirq.Gate]]:
+    """Used by deserialization code to convert string representations of gates to Cirq objects.
```

Comment:
Let's coordinate on how to make this docstring match with https://github.com/SupertechLabs/client-superstaq/pull/513/files#r1227291598 for consistency

Reply:
@vtomole I tried to pull from (IMO) the best of both of ours. PTAL.